icevision/visualize/show_data.py

```diff
@@ -67,6 +67,7 @@ def show_record(
 
 def show_pred(
     pred: Prediction,
+    class_map: Optional[ClassMap] = None,
```

Comment:
Heh, so the `class_map` traces on all show functions are actually a remnant of the latest refactor - they are not required anymore, the class_map is always inferred from the record, actually what we should do is remove this parameter from the functions haha

Reply:
ahh ok - good to know, would make sense too. good catch